index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>My resume</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+    <header>
+        <img src="img/profile_photo.jpg" height="69" width="70" alt="Profile Photo"/>
+        <p>Dzmitry Karachun<br><span>My profile</span></p>
+    </header>
+    <div class="container">
+        <div class="line_1">
```

Comment:
more meaningful class name should be used instead of "line_1" class (for ex. list_item, because in this case education, qualification - it's list of some points)
div tag can be replaced with li tag (in this case parent (div class="container") should be replaced by ul tag )